src/ui/public/timepicker/timepicker.less

```diff
@@ -8,7 +16,30 @@
   }
   .kbn-timepicker-section {
     float: left;
-    padding: 0px 15px;
+    padding-right: 15px;
```

Comment:
We can make each column the same width, and also remove the unnecessary padding from the last column by changing these styles:

```less
  .kbn-timepicker-section {
    float: left;
    width: 25%;

    & + .kbn-timepicker-section {
      padding-left: 15px;
    }
  }
```

![image](https://user-images.githubusercontent.com/1238659/26981678-bfdefb2c-4cea-11e7-9c2e-0dbf736f8b39.png)